pusher/src/Services/AdminApi.ts

```diff
@@ -141,6 +141,15 @@ class AdminApi {
             return data.data;
         });
     }
+
+    /*TODO add constant to use profile companny*/
+    getProfileUrl(accessToken: string): string {
+        if (!ADMIN_URL) {
```

Comment:
wait. ADMIN_URL is the deprecated version of API_URL.
I think _here_ you mean ADMIN_API_URL...